docs/coding/data-modeling.md

```diff
@@ -97,17 +97,21 @@ _asset scale_. For example, representing USD in cents uses an asset scale of `2`
 
 #### Examples
 
-- In USD, `$1` = `100` cents. So for example,
-  - The fractional amount `$0.45` is represented as the integer `45`.
-  - The fractional amount `$123.00` is represented as the integer `12300`.
-  - The fractional amount `$123.45` is represented as the integer `12345`.
+- `1 USD` = `100` cents. Using an asset scale of `2`,
+  - The fractional amount `0.45 USD` is represented as the integer `45`.
+  - The fractional amount `123.00 USD` is represented as the integer `12300`.
+  - The fractional amount `123.45 USD` is represented as the integer `12345`.
 
-### Oversized Amounts
+- `1 JPY` = `1` yen. Using an asset scale of `0`,
+  - The fractional amount `123 JPY` is represented as the integer `123`.
 
-The other direction works as well. If the smallest useful unit of a currency is `10,000,000` units,
-then it can be scaled down to the integer `1`.
+- `1 KWD` = `1000` fils. Using an asset scale of `3`,
+  - The fractional amount `0.450 KWD` is represented as the integer `450`.
+  - The fractional amount `123.000 KWD` is represented as the integer `123000`.
+  - The fractional amount `123.450 KWD` is represented as the integer `123450`.
 
-The 128-bit representation defines the precision, but not the scale.
+The other direction works as well. If the smallest useful unit of an asset is `10, 000, 000` units,
```

Comment:
FYI I personally don't like writing a number using a localized format such however I think in this context it's easiest for a reader to count the zeros if shown as `en_US`. I also considered zig format `1_000_000` since it's more international but I ultimately decided for documentation that may confuse a reader (this documentation is probably aimed at users of tigerbeetle, not developers of tigerbeetle)